lib/scala/project-manager/src/main/scala/org/enso/projectmanager/boot/Cli.scala

```diff
@@ -0,0 +1,56 @@
+package org.enso.projectmanager.boot
+
+import org.apache.commons.cli
+import org.enso.polyglot.LanguageInfo
+
+import scala.util.Try
+
+object Cli {
+
+  val JSON_OPTION    = "json"
+  val HELP_OPTION    = "help"
+  val VERBOSE_OPTION = "verbose"
+  val VERSION_OPTION = "version"
+
+  object option {
+
+    val help: cli.Option = cli.Option
+      .builder("h")
+      .longOpt(HELP_OPTION)
+      .desc("Displays this message.")
+      .build()
+
+    val verbose: cli.Option = cli.Option
+      .builder("v")
+      .longOpt(VERBOSE_OPTION)
+      .desc("Increase logs verbosity.")
+      .build()
```

Comment:
I had the impression that we shouldn't use short option names (but maybe this is an exception? definitely writing `--verbose --verbose` wouldn't be great).

If this option can indeed use the short name, I suggest the help text is made a bit more explanatory - what is the default verbosity level? Also from the current help text I'm not sure if I would guess that I can add `-v` twice to get to an even higher level (I know this is pretty common in many programs, but I think it should be explained).

Reply:
I think `-v` is fine for verbosity. I'll add extra info about options stacking :+1: